Source/WebCore/page/LocalFrameViewLayoutContext.h

```diff
@@ -128,6 +129,10 @@ class LocalFrameViewLayoutContext {
     UpdateScrollInfoAfterLayoutTransaction& updateScrollInfoAfterLayoutTransaction();
     UpdateScrollInfoAfterLayoutTransaction* updateScrollInfoAfterLayoutTransactionIfExists() { return m_updateScrollInfoAfterLayoutTransaction.get(); }
 
+    void markNeedingTransformUpdate(RenderLayerModelObject* parent, RenderLayerModelObject* descendant);
```

Comment:
`parent` isn't really accurate; it's about containing block.

Reply:
Fixed.